mod/groups/views/default/forms/groups/invite.php

```diff
@@ -8,7 +8,7 @@
 $group = $vars['entity'];
 $owner = $group->getOwnerEntity();
 $forward_url = $group->getURL();
-$friends = elgg_get_logged_in_user_entity()->getFriends('', 0);
+$friends = elgg_get_logged_in_user_entity()->getFriends(array(''), 0);
```

Comment:
I bet the 0 was for limit. If so you'd want an array with "limit" => 0. No 2nd argument.